src/main/java/mods/eln/gui/ItemStackFilter.java

```diff
@@ -51,7 +51,7 @@ public static ItemStackFilter[] OreDict(String name) {
     public boolean tryItemStack(ItemStack itemStack) {// caca1.5.1
         if (Utils.getItemId(itemStack) != itemId)
             return false;
-        if ((itemStack.getItemDamage() & damageMask) != damageValue)
```

Comment:
> I don't think this is right, because the damage mask could be used to allow groups of items. This "fixes" it but it isn't the correct fix. The probable issue is that other mod's coal uses damage values that are not the correct ones and the masks don't line up. The real fix I think is to set the damage mask correctly to allow any damage values.

(Machine translation)
Thank you for your reply.  My code is too crude and didn't take other mods into account, so my solution is not a correct one, yours should be a correct one